packages/ui-file-drop/src/FileDrop/utils/accepts.ts

```diff
@@ -45,7 +45,7 @@ function accepts(file: any, acceptProp: any) {
   return true
```

Comment:
`return acceptList.some((type: any) => {` -> `return acceptList.some((type) => {`
TS will infer (figure out) the correct type for this